### PR TITLE
Mongo cache collaboration

### PR DESCRIPTION
**Pull request by mehmetalisavas**
Redis has been removed with this PR.
Also we will use new mongo cache system instead of using Redis in collaboration.

Before merging this PR, https://github.com/koding/koding/pull/10092 needs to be merged.

### CODE REVIEW

File: go/src/socialapi/workers/collaboration/collaboration.go [hunk 1]
```diff
@@ -169,30 +169,30 @@ func (c *Controller) checkIfKeyIsValid(ping *models.Ping) (err error) {
...
-
-	if err == redis.ErrNil {
-		c.log.Debug("redis key not found %+v", ping)
+	if err == cache.ErrNotFound {
+		c.log.Debug("key is not found %+v", ping)
 		return errSessionInvalid // key is not there
 	}
 
-	unixSec, err := strconv.ParseInt(pingTime, 10, 64)
+	pt := fmt.Sprint(pingTime)
```
Comment (cihangir) [inline go/src/socialapi/workers/collaboration/collaboration.go]: Sprint uses reflection, you can just do type casting here

**Review by cihangir — APPROVED** [review]
LTGM, good bye redis!

**Comment by cihangir** [comment]
other pr is merged, could you rebase from master?

**Comment by mehmetalisavas** [comment]
@cihangir PTAL

**Review by cihangir — APPROVED** [review]

**Review by cihangir — APPROVED** [review]

**Pull request merged**